sql-cli/pyproject.toml

```diff
@@ -34,6 +36,10 @@ tests = [
     "pytest-cov",
     "mypy"
 ]
+all = [
+    "apache-airflow>=2.0",
+    "astro-sdk-python[all]>=1.0.0",
+]
```

Comment:
I have added these two as optional dependencies. For example, they are already useful to validate the syntax of the generated DAG.

I would suggest we make them required dependencies when we have `flow run`.